Mathlib/GroupTheory/SpecificGroups/BraidGroup/Basic.lean

```diff
@@ -0,0 +1,82 @@
+/-
+Copyright (c) 2024 Hannah Fechtner. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Hannah Fechtner
+-/
+import Mathlib.GroupTheory.PresentedGroup
+import Mathlib.Data.Nat.Dist
+
+/-!
+#  Braid Groups
+
+Artin's braid group on infintely many strands
+
+## Main definitions
+
+* `BraidGroupInf ` is the braid group on infinitely many strands, defined as a PresentedGroup
+  on the braid relations
+
+* `braid_group n` is the braid group on n+1 strands (thus n generators) - coming in the next PR
+
+## Main results
+* none in this PR
+
+
+## Tags
+braid group
+
+## TODO
+
+-/
+
+namespace BraidGroup
+
+/-- the set of relations on FreeGroup ℕ corresponding to vaild braid moves -/
```

Comment:
Please use the docstring to state the relations in a notation that is maybe a bit easier to read than the Lean notation with all the `.of`s.